frontend/src/types.ts

```diff
@@ -277,6 +282,11 @@ export type TrackingEventProperties = {
   projectName?: string;
   notebookName?: string;
   lastActivity?: string;
+  href?: string;
+  replicaCount?: number;
+  outcome?: TrackingOutcome; // TODO better name for key?
```

Comment:
This key is to denominate how a Modal / wizard was left (clicking submit or cancelling early). It should get a better name than 'outcome'. Perhaps 'wayOut' ? That name change would also impact the type a few lines above